tests/functional/test_install_cleanup.py

```diff
@@ -30,7 +30,7 @@ def test_no_clean_option_blocks_cleaning_after_install(script, data):
     build = script.base_path / 'pip-build'
     script.pip(
         'install', '--no-clean', '--no-index', '--build', build,
-        '--find-links=%s' % data.find_links, 'simple',
+        '--find-links=%s' % data.find_links, 'simple', expect_temp=True,
```

Comment:
The cache should be cleaning up after itself even if this happens?

Reply:
This was controlled by the `no_clean` option but I have disabled that behavior.